app/init.php

```diff
@@ -127,6 +127,7 @@
  */
 Config::load('events', __DIR__.'/config/events.php');
 Config::load('auth', __DIR__.'/config/auth.php');
+Config::load('errorCodes', __DIR__.'/config/errorCodes.php');
```

Comment:
```suggestion
Config::load('errors', __DIR__.'/config/errors.php');
```

Prefer a single name when possible.